src/App.tsx

```diff
@@ -96,15 +96,19 @@ export default function App() {
       Buffer.from(encryptedMessageHex, 'hex'),
     );
 
-    const decryptedMessage = await decrypt(
-      provider,
-      domain,
-      encryptedMessage,
-      'https://porter-tapir.nucypher.io/',
-      provider.getSigner(),
-    );
+    try {
+      const decryptedMessage = await decrypt(
```

Comment:
This demo is using an older version of the API - https://github.com/nucypher/taco-ardrive-demo/blob/master/package.json#L17. Let's update to the latest version, `@nucypher/taco` `0.5.0`, and update the call to `decrypt` appropriately. You can use the code from here as an example - https://github.com/nucypher/taco-web/blob/main/examples/taco/nextjs/src/hooks/useTaco.ts#L36.

Reply:
✔️